functions.yml

```diff
@@ -89,8 +94,9 @@ complex_type_manipulation:
       args:
         - name: struct
           doc: The struct to expand
+      returns: Columns corresponding to fields of the input struct
     - name: explode_matrix
-      doc: Explode a spark.ml Matrix into arrays of rows
+      doc: Explodes a ``spark.ml`` matrix (sparse or dense) into multiple arrays, one per row of the matrix.
```

Comment:
For consistency, we should write out `spark.ml` `Matrix`; we don't need to specify that it's sparse/dense as those are the only two implementation (`SparseMatrix` and `DenseMatrix`)

Reply:
changed to ``Matrix``. Left sparse or dense for emphasis